packages/block-library/src/columns/editor.scss

```diff
@@ -89,13 +89,17 @@
 
 			// Beyond mobile, allow 2 columns.
 			@include break-small() {
-				flex-basis: 50%;
+				// Size should be half the available width, minus the margins added below
```

Comment:
@tofumatt has taught me to be consistent (going forward, I was a bad actor for a long time) with our code comment system. Which is capital first letter, end with a period.

In this case we're missing a period at the end.